crypto/provider_core.c

```diff
@@ -680,6 +680,11 @@ int ossl_provider_activate(OSSL_PROVIDER *prov)
     return 0;
 }
 
+int ossl_provider_activate_leave_fallbacks(OSSL_PROVIDER *prov)
```

Comment:
This does not need to exist - just use OSSL_PROVIDER_try_load and this isn't required - as it was only added for OSSL_PROVIDER_load_leave_fallbacks